concourse/pipelines/rhui-release.jsonnet

```diff
@@ -0,0 +1,142 @@
+local project = 'google.com:rhel-infra';
+
+local gatejob = {
+  local job = self,
+
+  name: error 'must set name on gatejob',
+  trigger:: true,
+  passed:: [],
+  plan: [
+    {
+      get: imageResource,
+      trigger: job.trigger,
+      passed: job.passed,
+    }
+    for imageResource in ['cds-image', 'rhua-image']
+  ],
+};
+
+local gcloudmigtask = {
+  local task = self,
+
+  node:: error 'must set node on gcloudmigtask',
+  stage:: 'prod',
+  region:: error 'must set region on gcloudmigtask',
+  action:: error 'must set action on gcloudmigtask',
+
+  task: error 'must set task',
+  config: {
+    image_resource: {
+      source: {
+        repository: 'google/cloud-sdk',
+        tag: 'alpine',
+      },
+      type: 'registry-image',
+    },
+    platform: 'linux',
+    run: {
+      args: [
+        'compute',
+        'instance-groups',
+        'managed',
+      ] + task.action + [
+        '--quiet',
+        '%s-mig-%s-%s' % [task.node, task.stage, task.region],
+        '--region=' + task.region,
+        '--project=' + project,
+      ],
+      path: 'gcloud',
+    },
+  },
+};
+
+local deployjob = gatejob {
+  local job = self,
+
+  stage:: 'prod',
+  region:: error 'must set region on deployjob',
+  plan: super.plan + std.flattenArrays([
```

Comment:
@hopkiw -- I know we talked about avoiding functions from `std` :)  The goal is to repeat the three tasks for each node type (the tasks being update, wait for version, wait for stable), and this was the only way I could think of to achieve that, since as far as I can tell jsonnet's object comprehension only allows a single object to be iterated.